docs/ANDROID.md

```diff
@@ -2,11 +2,32 @@
 
 ## Building Hermes from source
 
-Because we're using a version of Hermes patched with Node-API support, we need to build React Native from source.
+### Step 1: `settings.gradle`
 
+Gradle needs specific configuration to build React Native from source: dependency substitutions for React Native & Hermes modules, and modifications to default logic in RN build scripts, which is done by setting an environment variable, as described in this section.
+
+In your app's `settings.gradle` please include the below:
+
+```groovy
+// customize the path to match your node_modules location
+apply(from: "../../../node_modules/react-native-node-api/android/app-settings.gradle")
```

Comment:
This path (`../../../node_modules`) assumes a monorepo structure, so won't work on a regular repo.

Can we use proper Node module resolution instead? As demonstrated [here](https://github.com/expo/expo/blob/ab9f960751807743c596f80b56e698118d1aeb53/templates/expo-template-bare-minimum/android/app/build.gradle#L13).